### PR TITLE
project-movies-julia-helena

### PR DESCRIPTION
**Pull request by HelenaW86**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/Components/Loading/Loading.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        </span>
+      </h1>
+      <div className="ring">
+        Loading
+        <span className="spinner"></span>
+      </div>
+    </>
+  );
+};
+export default Loading;
```
Comment (themisk84) [inline code/src/Components/Loading/Loading.js:21]: A very nice loader!

File: code/src/pages/MovieList/MovieList.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            />
+            <div className="movie-title-date">
+              <h1>{movie.title}</h1>
+              <p className="date-movie">
+                Released: {moment(movie.release_date).format("LL")}
+              </p>
+            </div>
+          </Link>
+        ))}
+      </main>
```
Comment (themisk84) [inline code/src/pages/MovieList/MovieList.js:31]: Excellent job!

File: code/src/Components/Loading/Loading.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import "./Loading.css";
+
+const Loading = () => {
+  return (
+    <>
+      <h1 className="welcome-message">
```
Comment (mamite100) [inline code/src/Components/Loading/Loading.js:8]: Realy nice loader page, with camera emoji and everything

**Review by themisk84 — APPROVED** [review]
Awesome job Helena and Julia!You challenged yourselves with all the black level targets and you made it!Keep up the good work!

File: code/src/pages/MovieList/MovieList.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import Dropdown from "Components/Dropdown/Dropdown";
+import "./MovieList.css";
+
+const MovieList = ({ select, setSelect, movieList, loading }) => {
+  return (
+    <>
+      <main className="movie-list-container">
+        <Dropdown select={select} setSelect={setSelect} />
+        {movieList.map((movie) => (
+          <Link
```
Comment (mamite100) [inline code/src/pages/MovieList/MovieList.js:13]: Very nice function to link the hole movie card. Nice code lines

File: code/src/Components/Dropdown/Dropdown.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import "./Dropdown.css"
+
+const Dropdown = ({ select, setSelect }) => {
```
Comment (mamite100) [inline code/src/Components/Dropdown/Dropdown.js:4]: Very very nice with the drop down and the Selct

File: code/src/Components/Loading/Loading.css [hunk 6]
```diff
@@ -0,0 +1,97 @@
+body {
```
Comment (mamite100) [inline code/src/Components/Loading/Loading.css:1]: Realy nice style of the loading page! The code is clean end easy to read

File: code/src/pages/Characters/Characters.css [hunk 7]
```diff
@@ -0,0 +1,69 @@
+.characters-list-container {
```
Comment (mamite100) [inline code/src/pages/Characters/Characters.css:1]: impressive!! well done!

**Review by mamite100 — APPROVED** [review]

**Pull request closed without merging**